packages/backend/package.json

```diff
@@ -125,7 +125,7 @@
     "vitest": "^3.1.2"
   },
   "dependencies": {
-    "@crc-org/macadam.js": "0.0.1-202504301306-5ade487",
+    "@crc-org/macadam.js": "0.0.1-202505091153-d597ec3",
```

Comment:
is the update needed to be in sync with the workflow update?

Reply:
unfortunately yes as it's now published to npmjs and not github packages (so we don't have previous versions)

but for the approval, need to see if it's still working with this new version

Reply:
Right now it needs to be in sync with rhel extension version too 😕, but agreed, one step at a time.

Reply:
I made a PR in rhel extension as well